test/postServiceTest.js

```diff
@@ -0,0 +1,19 @@
+const mocha = require('mocha');
+const sinon = require('sinon');
+const assert = require('assert');
+const Post = require('../models/Post');
+const mongoose = require('../mongoose');
```

Comment:
unused variable. Please run eslint before you submit a review

Reply:
fixed